### PR TITLE
Bug 1209387 - Notification container scrolling in native-scrolling Ut…

### PR DESCRIPTION
**Pull request by mcav**
…ilityTray causes tray to fling up due to nested scroll container momentum.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cb7f43b70a53ab2e4e807ffd0902f01295bff5cf

File: apps/system/index.html [hunk 1]
```diff
@@ -253,48 +253,50 @@
...
+                </div>
+                <!-- Emergency callback manager -->
+                <div id="emergency-callback-notification" class="fake-notification" hidden role="link">
+                  <div data-icon="call-emergency" class="alert" aria-hidden="true"></div>
+                  <div class="title-container" data-l10n-id="emergency-callback-mode"></div>
+                  <div class="timer detail"></div>
+                </div>
+                <!-- Storage watcher notification -->
+                <div id="storage-watcher-container" class="fake-notification" role="link">
+                  <div data-icon="storage-circle" aria-hidden="true"></div>
```
Comment (mcav) [inline apps/system/index.html]: note: the diff looks weird here, but it was just a straightforward wrap+indent

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=17ce89470dc03fc368516ce2af5db2e000da16fd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=25dc74d984744ff5913937d04fb6a4da5824dd2a

File: apps/system/index.html (hunk 1 above)
Comment (etiennesegonzac) [inline apps/system/index.html]: `?w=1` to the rescue!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d05c0e82764bdfab7d97df18d59b762e9acca6b2

**Pull request merged**